portfolio/src/main/webapp/script.js

```diff
@@ -11,55 +11,57 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
+ 
 function addRandomGreeting() {
   const greetings =
       ['I am 20 years old', 'My first language is French', 'I have a sister', 
       'My favorite band is Pink Floyd', 'I am half Persian'];
-
-  // Pick a random greeting.
+ 
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
+ 
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-
-
+ 
+ 
 function randomizeImage() {
-  // The images directory contains 13 images, so generate a random index between
-  // 1 and 13.
   const imageIndex = Math.floor(Math.random() * 7) + 1;
   const imgUrl = 'images/IMG_' + imageIndex + '.jpeg';
-
+ 
   const imgElement = document.createElement('img');
   imgElement.src = imgUrl; 
-
+ 
   var imageContainer = document.getElementById('random-image-container');
-  // Remove the previous image.
+
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
-
+ 
 function showFrame() {
     document.getElementById("showiFrame").innerHTML = "<iframe src=\"https://open.spotify.com/embed/track/1bSpwPhAxZwlR2enJJsv7U\" width=\"300\" height=\"380\" frameborder=\"0\" allowtransparency=\"true\" allow=\"encrypted-media\" ></iframe>";
-
-
 }
 
-function getDataUsingArrowFunctions() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
-  });
-}
+function getData() {
+  fetch('/data').then(response => response.json()).then((data) => {
 
+    const comments = document.getElementById('comments-container');
```

Comment:
nit: might be better to rename `data` to `comments` and `comments` to `commentList` or something similar. See https://drive.google.com/file/d/1-VB6otFQRR2SOESwAXRCHiHiZ4tx11iw/view